### PR TITLE
Add TESTING.md with Detailed Manual Test Cases

### PR DESCRIPTION
**Pull request by CFIALeronB**
Created TESTING.md to document manual testing procedures for key functionalities, including testing for Capture, Switch, Classify, Load, Save, Directories, and Results components. Each test case outlines the objective, preconditions, steps, expected results, and pass/fail criteria, providing a comprehensive guide for ensuring application reliability.

#67

### CODE REVIEW

**Review by rngadam — CHANGES_REQUESTED** [review]
Thanks for doing this! combined with doc in dev-rel-docs, this will be good testing discipline.

Suggestions:

1) we'll need guidelines on how to execute manual testing. I think along the line of "create an issue references this specific commit hash of TESTING.md, copy the content in the description of the issue, review each checklist, report results and for failing cases create new issues"

2) the numbering of the issue; I'd suggest numbering by referencing the issue number that created the feature with probably a sub for when many tests are done for the same issue. example:

Test Issue #1 - A

File: TESTING.md [hunk 1]
```diff
@@ -2,6 +2,33 @@
 
 This document provides detailed instructions and procedures for manually testing the various functionalities of Nachet, ensuring that all features operate correctly and as expected before deployment or release.
 
+## Test Case: Backend URL Warning Functionality
+
+**Objective:** Verify the application correctly displays a warning if the backend URL is not set or is non-responsive.
+
+**Preconditions:**
+- [ ] The application is open.
+- [ ] Nachet Backend is set up and running locally. The Nachet Backend can be found at [Nachet Backend GitHub Repository](https://github.com/ai-cfia/nachet-backend). To run the application locally, use the command `hypercorn -b :8080 app:app` in the terminal.
```
Comment (rngadam) [inline TESTING.md:11]: >80

**Review by rngadam — CHANGES_REQUESTED** [review]
linewrap documentation

File: TESTING.md (hunk 1 above)
Comment (CFIALeronB) [reply TESTING.md:11]: what does this mean @rngadam?

Comment (rngadam) [reply TESTING.md:11]: means the document is not wrapping at column 80

**Review by rngadam — APPROVED** [review]

**Pull request merged**